### PR TITLE
Project development and modification

### PR DESCRIPTION
**Pull request by Lancelot-SO**
-Added the contact section and footer section to the index.html file to finish the mobile version of the project.
-Styled the contact and footer sections as required.
-Configured linters to work perfectly in the project and therefore there were no errors.
-Modified the ReadMe.md file to suit the project requirement and give a better description of the project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -361,5 +361,36 @@ <h4>Skills</h4>
...
 
+    <section class="contact">
+      <div class="contact-text">
+        <p>
+          I'm always interested in hearing about new projects, 
+          so if you'd like to chat please get in touch.
+        </p>
+      </div>
+
+      <form action="" class="contact-form" method="post">
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [REQUIRED] Good job so far! `Form` should use the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html), `action` value should be similar to `https://formspree.io/f/{form_id}` but you forgot it implement to your **form's** `action`. Please implement [formspree service](https://formspree.io/html) 🙏🏼 

image

File: index.html [hunk 2]
```diff
@@ -361,5 +361,36 @@ <h4>Skills</h4>
...
+
+    <footer>
+      <div class="footer-line"></div>
+      <div class="footer-socials">
+        <ul>
+          <li><a href="#"><img src="assets/Group.png" alt=""/></a></li>
+          <li><a href="#"><img src="assets/Vector.png" alt=""/></a></li>
+          <li><a href="#"><img src="assets/Vector2.png" alt=""/></a></li>
+          <li><a href="#"><img src="assets/Vector3.png" alt=""/></a></li>
+          <li><a href="#"><img src="assets/Vector4.png" alt=""/></a></li>
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [REQUIRED] Good job so far! Please change the color of your social network icons to require [Fgma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=mUwfyaAzix1prFLg-0) color (`#505F79`) 🙏🏼 

image

File: index.html [hunk 3]
```diff
@@ -361,5 +361,36 @@ <h4>Skills</h4>
...
+        <p>
+          I'm always interested in hearing about new projects, 
+          so if you'd like to chat please get in touch.
+        </p>
+      </div>
+
+      <form action="" class="contact-form" method="post">
+        <input type="text" name="name" class="contact-form-text" maxlength="30" placeholder="Full name" required>
+        <input type="email" name="email" class="contact-form-email" placeholder="Email address" required>
+        <textarea name="contact" name="message" class="contact-textarea" maxlength="500" placeholder="Enter your text here" required></textarea>
```
Comment (mukhammadrizooff) [inline index.html:375]: - [ ] [OPTIONAL] Good job so far! If you remove one of the `name` attributes on your `<textarea>` it would be clean 👍🏼

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Lancelot-SO, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
### Hi  @Lancelot-SO   , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve

## Highlights

GitHub review feature :heavy_check_mark:
Inline comment :heavy_check_mark: 
The comments are given in a friendly and polite manner :heavy_check_mark: 
Explain why changes were suggested :heavy_check_mark: 

## Optional 

- [ ] [**OPTIONAL**] _**While you make great  effort on this project , I check the commit history to make sure that everybody has contributed code on each project. However, your teammate didn't. It's crucial to have the contribution of all team members in such as project. Next time make sure you have the contribution of everybody in the PR. If it was the first review I should invalidate your project.**_

**_Every comment with the [**OPTIONAL**] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@iambenkis` in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**